Bundle/CoreBundle/Entity/View.php

```diff
@@ -696,9 +735,10 @@ public function getReference()
      *
      * @return $this
      */
-    public function setReference($reference)
+    public function setReference(ViewReference $reference, $locale = null)
     {
-        $this->reference = $reference;
+        $locale = $locale ?: $this->getLocale();
```

Comment:
The Object ViewReference already has a local property, is it necessary to force the locale?